3444/0/triangle.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+
+# :reek:UtilityFunction and :reek:FeatureEnvy
+def first_ex(a_side, b_side, c_side)
+  (a_side + b_side) <= c_side || (b_side + c_side) <= a_side || (c_side + a_side) <= b_side
```

Comment:
parentheses are not needed here actually. You can also make a method which will check the inequality for a single permutation

Reply:
sorry, but i can't correct understand what u mean under ' method which will check the inequality for a single permutation'. I guess i need to add more abstraction?

Reply:
kinda
```
def satisfies_triangle_inequality?(side_a, side_b, side_c)
  side_a + side_b > side_c
end
```